packages/services/storage/src/index.ts

```diff
@@ -477,64 +426,55 @@ export async function createStorage(
       { superTokensUserId }: { superTokensUserId: string },
       connection: Connection,
     ) {
-      const user = await connection.maybeOne<
-        users & {
-          provider: string | null;
-        }
-      >(sql`/* getUserBySuperTokenId */
+      const record = await connection.maybeOne<unknown>(sql`/* getUserBySuperTokenId */
         SELECT
-          u.*,
-          stu.third_party_id as provider
+          ${userFields(sql`"users".`, sql`"stu".`)}
         FROM
-          users as u
-        LEFT JOIN supertokens_thirdparty_users as stu ON (stu.user_id = u.supertoken_user_id)
+          "users"
+        LEFT JOIN "supertokens_thirdparty_users" AS "stu"
+          ON ("stu"."user_id" = "users"."supertoken_user_id")
         WHERE
-          u.supertoken_user_id = ${superTokensUserId}
+          "users"."supertoken_user_id" = ${superTokensUserId}
         LIMIT 1
       `);
 
-      if (user) {
-        return transformUser(user);
+      if (!record) {
+        return null;
       }
 
-      return null;
+      return UserModel.parse(record);
     },
     async createUser(
       {
         superTokensUserId,
         email,
         fullName,
         displayName,
-        externalAuthUserId,
         oidcIntegrationId,
       }: {
         superTokensUserId: string;
         email: string;
         fullName: string;
         displayName: string;
-        externalAuthUserId: string | null;
         oidcIntegrationId: string | null;
       },
       connection: Connection,
     ) {
-      const user = await connection.one<users>(
+      await connection.query<unknown>(
```

Comment:
why `unknown`? 🤔

Reply:
because we don't want anyone to use that 🗡️